pkg/controller/backup-operator/sync.go

```diff
@@ -109,12 +110,23 @@ func (b *Backup) handleErr(err error, key interface{}) {
 	b.logger.Infof("Dropping etcd backup (%v) out of the queue: %v", key, err)
 }
 
-func (b *Backup) handleBackup(spec *api.BackupSpec) error {
+type backupStatus struct {
```

Comment:
Don't redefine this.
Just reuse api.BackupCRStatus, ignore the succeeded and Reason fields

Reply:
done.!